lang/en/admin/server.php

```diff
@@ -36,6 +36,7 @@
     'disabled' => 'Disabled',
     'memory' => 'Memory',
     'memory_limit' => 'Memory Limit',
+    'memory_helper' => 'Wings will add overhead to this value when creating the container to make sure it doesn\'t starve out when using max memory.',
```

Comment:
Rewrite

```suggestion
    'memory_helper' => 'While the Server is being created, additional memory will be temporarily used to help prevent issues.',
```

Reply:
That description is wrong. The overhead is added to the normal container, not the install container.